chromium_src/components/content_settings/core/browser/content_settings_registry.cc

```diff
@@ -228,6 +228,18 @@ void ContentSettingsRegistry::BraveInit() {
            ContentSettingsInfo::INHERIT_IF_LESS_PERMISSIVE,
            ContentSettingsInfo::EXCEPTIONS_ON_SECURE_AND_INSECURE_ORIGINS);
 
+  // Register AI chat permission default value as Ask.
+  Register(ContentSettingsType::BRAVE_AI_CHAT, "brave_ai_chat",
+           CONTENT_SETTING_ASK, WebsiteSettingsInfo::UNSYNCABLE,
+           /*allowlisted_schemes=*/{},
+           /*valid_settings=*/
+           {CONTENT_SETTING_ALLOW, CONTENT_SETTING_BLOCK, CONTENT_SETTING_ASK},
+           WebsiteSettingsInfo::TOP_ORIGIN_ONLY_SCOPE,
+           WebsiteSettingsRegistry::DESKTOP |
+               WebsiteSettingsRegistry::PLATFORM_ANDROID,
+           ContentSettingsInfo::INHERIT_IF_LESS_PERMISSIVE,
+           ContentSettingsInfo::EXCEPTIONS_ON_SECURE_AND_INSECURE_ORIGINS);
```

Comment:
we probably want it to only be set on secure origins (if we ever allow it to be set on anything other than search.brave.com), so [EXCEPTIONS_ON_SECURE_ORIGINS_ONLY](https://source.chromium.org/chromium/chromium/src/+/main:components/content_settings/core/browser/content_settings_info.h;drc=08efa89c7d73d72e6ebcddbf053c41a230dc1ba8;bpv=1;bpt=1;l=43?gsn=EXCEPTIONS_ON_SECURE_ORIGINS_ONLY&gs=KYTHE%3A%2F%2Fkythe%3A%2F%2Fchromium.googlesource.com%2Fcodesearch%2Fchromium%2Fsrc%2F%2Fmain%3Flang%3Dc%252B%252B%3Fpath%3Dcomponents%2Fcontent_settings%2Fcore%2Fbrowser%2Fcontent_settings_info.h%234taIkN86tsPWlhLzLhwgjiF4zPj1luXCdharFqy-3Ss) seems appropriate

Reply:
fixed in 52a60130f4680ac6b22955206ff08c0ad47d127a